### PR TITLE
OSSMDOC-594: update remove SMCP procedure to include SMRR

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-594

Link to docs preview:
https://deploy-preview-45826--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/removing-ossm.html#ossm-control-plane-remove-cli_removing-ossm

Additional information:
Eng review - jwendell
QE review - pbajjuri20
Peer review - jstickler

### CODE REVIEW

File: modules/ossm-control-plane-remove.adoc [hunk 1]
```diff
@@ -39,6 +39,20 @@ You can remove the {SMProductName} control plane by using the CLI.  In this exam
 
 . Log in to the {product-title} CLI.
 
+. Run this command to retrieve the name of the installed `ServiceMeshMemberRoll`:
++
+[source,terminal]
+----
+$ oc get smrr -n istio-system
```
Comment (jwendell) [inline modules/ossm-control-plane-remove.adoc:46]: typo: smmr

Comment (jwendell) [reply modules/ossm-control-plane-remove.adoc:46]: By the way, just noticed that we don't need this step as the recource name is always `default`. So we can remove this step and just document the next one, with a hardcoded resource name, e.g.

```
oc delete smmr default -n istio-system
```

Comment (rh-tokeefe) [reply modules/ossm-control-plane-remove.adoc]: @jwendell thanks for chatting. I have updated the PR and squashed the job.

**Review by jwendell — APPROVED** [review]

**Comment by pbajjuri20** [comment]
LGTM..!

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**